pkg/cmd/build/logger.go

```diff
@@ -54,7 +54,47 @@ func deployDisplayer(ctx context.Context, ch chan *client.SolveStatus) error {
 					oktetoLog.Info(err.Error())
 					continue
 				}
-				t.display()
+				t.displayDeploy()
+				t.removeCompletedSteps()
+			} else {
+				done = true
+			}
+			if done {
+				oktetoLog.StopSpinner()
+				if t.err != nil {
+					return t.err
+				}
+				return nil
+			}
+		}
+	}
+}
+
+func destroyDisplayer(ctx context.Context, ch chan *client.SolveStatus) error {
```

Comment:
I think the only thing that change is the spinner text. Could we pass an argument and reuse the function to not have duplicated code (whenever we need to update one we'll need to update the other and it's very likely that we forget 😅 ).

Reply:
Here's what I've understood up till now:

- I think in the following code:

```buildOptions := build.OptsFromBuildInfoForRemoteDeploy(buildInfo, &types.BuildOptions{OutputMode: "deploy"})```

When we pass the _OutputMode_ as "deploy" or "destroy", this leads to the case we've mentioned in the switch statement in _buildkit.go_:

```
case "deploy", "destroy":
		err := deployDisplayer(context.TODO(), plainChannel)
		commandFailChannel <- err
		return err
```

 - The _deployDisplayer()_ method at [L:33](https://github.com/okteto/okteto/blob/master/pkg/cmd/build/logger.go#L33) calls [t.display() ](https://github.com/okteto/okteto/blob/master/pkg/cmd/build/logger.go#LL135C7-L135C7) method at L:135 that is responsible for displaying the text which we need to modify. 
 
 Correct?
 
 **Problem that I'm facing:**
 
 - I'm thinking of adding an _if_ statement to check about which cmd is being executed - whether it's "destroy" or "deploy". I think, one way of doing this is by fetching the value from [here](https://github.com/okteto/okteto/blob/master/cmd/destroy/remote.go#L149). However, this is not directly possible to access its value, how should I proceed to fetch that value? 
 
- If we were to pass an argument, what should it look like?

Reply:
Sorry I didn't see this message. Everthing you posted is correct. 

> I'm thinking of adding an if statement to check about which cmd is being executed - whether it's "destroy" or "deploy". I think, one way of doing this is by fetching the value from [here](https://github.com/okteto/okteto/blob/master/cmd/destroy/remote.go#L149). However, this is not directly possible to access its value, how should I proceed to fetch that value?

You can add a new argument to `displayer` the `progress` variable which contains `deploy` or `destroy`

I hope that answer your question if not let me know

Reply:
I also believe that adding an argument is the cleanest approach in this case. I see that there are details still to be covered in the PR. If you don't mind I will change it to draft to consider it's ready for review status and make a new revision. 👍

Reply:
- Should the type of `progress` variable be `*types.BuildOptions`?
- Just to confirm, do we intend to add the `progress` argument to `t.display()` func, right?

Reply:
> Should the type of progress variable be *types.BuildOptions?

I think we can work with it being a string since it's the only thing we need from `*types.BuildOptions`

> Just to confirm, do we intend to add the progress argument to t.display() func, right?

Yes, we should add the argument there

Reply:
Fixed!